### PR TITLE
Remove commons lang

### PR DESCRIPTION
**Pull request by PascalSchumacher**
objenesis 2.3 contains some bugfixes releated to android support: https://github.com/easymock/objenesis/commit/3938c80acc5b60cc4550f493fe4d0a882a367a36

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -180,11 +179,6 @@
                 <version>${jackson.version}</version>
             </dependency>
             <dependency>
-                <groupId>org.apache.commons</groupId>
```
Comment (benas) [inline pom.xml]: Thanks, I forgot that. Looks like I'm tired..

**Comment by benas** [comment]
> objenesis 2.3 contains some bugfixes releated to android support: easymock/objenesis@3938c80

I hope this will fix #62 ! I'll give it a try tomorrow

**Comment by PascalSchumacher** [comment]
Not sure it will help with #62, java8 support of android is still limited: http://developer.android.com/preview/j8-jack.html

**Comment by benas** [comment]
Indeed, it could be the reason behind issue #62 . Looks like this issue will wait a little bit ..

Anyway, thank you for this PR!

**Pull request merged**